src/components/Filters/FilterRangeSlider.tsx

```diff
@@ -118,6 +118,8 @@ export default function FilterRangeSlider(props: FilterRangeSliderProps) {
       setInputMaxValue(selectedValue);
       modifyRequestRangeFilter(newRange);
     }
+
+    setIsModified(true);
```

Comment:
This prevents the initial `useEffect` function from rerendering to the default values